code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -264,25 +261,21 @@ stare
 		if(istype(reserved) && !isfeathertile(reserved))
 			. = get_path_to(holder.owner, reserved, max_dist, 1)
 			if(length(.)) //if we got a valid path
-				return
+				return reserved //don't like this - TODO can I make it better? just need to fix reserved tiles - maybe they could expire?
```

Comment:
![image](https://user-images.githubusercontent.com/65367576/200833847-37e34af2-7103-4069-ad32-35225cec3eac.png)
Aren't reservations mostly related to tasks? What if they just get wiped when the task switches?

Reply:
that could work